template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/batch_inference/notebooks/BatchInference.py.tmpl

```diff
@@ -26,12 +26,21 @@ dbutils.widgets.dropdown("env", "dev", ["dev", "staging", "prod"], "Environment
 dbutils.widgets.text("input_table_name", "", label="Input Table Name")
 # Delta table to store the output predictions.
 dbutils.widgets.text("output_table_name", "", label="Output Table Name")
+{{- if (eq .input_include_models_in_unity_catalog "no") }}
 # Batch inference model name
-dbutils.widgets.text("model_name", "", label="Model Name")
+dbutils.widgets.text(
+    "model_name", "dev-{{template `model_name` .}}", label="Model Name"
+)
+{{else}}
+# Unity Catalog registered model name to use for the trained mode.
+dbutils.widgets.text(
+    "model_name", "dev.{{template `schema_name` .}}.{{template `model_name` .}}", label="Model Name"
```

Comment:
I wonder if we want to hardcode dev here, should we use the target instead or is that risky?

Reply:
Yeah but the user may be using a different name for their `dev` catalog/target e.g. `development`. Isn't it safer to just pull this as well?

Reply:
but if running the notebook interactively you wouldn't have the target variable?

Reply:
Yeah that's fair, can we make it a `dbutils.widget` then that defaults to `dev` so it would be easier for the user to see the variable/specify it interactively?